src/Grid/GridDay.vala

```diff
@@ -101,8 +101,10 @@ public class Maya.View.GridDay : Gtk.EventBox {
         Gtk.TargetEntry dnd = {"binary/calendar", 0, 0};
         Gtk.drag_dest_set (this, Gtk.DestDefaults.MOTION, {dnd}, Gdk.DragAction.MOVE);
 
-        this.notify["date"].connect (() => {
-            label.label = date.get_day_of_month ().to_string ();
+        this.bind_property ("date", label, "label", BindingFlags.SYNC_CREATE, (binding, srcval, ref targetval) => {
+            DateTime date = (DateTime) srcval;
```

Comment:
```suggestion
            unowned var date = (GLib.DateTime) srcval.get_boxed ();
```
I know that the GValue cast is automatic in Vala but I find explicit getter safer here